src/operator/tensor/indexing_op.cc

```diff
@@ -52,7 +88,7 @@ NNVM_REGISTER_OP(_backward_Embedding)
 
 
 NNVM_REGISTER_OP(take)
-.describe(R"code(Takes elements from an input array along the given axis.
+.describe(R"code(Takes elements from an input array according to the input indices along the given axis.
```

Comment:
"takes elements ... according to the input indices" is a weird phrase. Please make clearer.